### ISSUE TITLE
BT Trackers Not Working

### ISSUE THREAD

**Issue by themusicgod1** [Tribler/tribler#2919]
#### Please, fill all relevant items: ####

- [x] I have read contributing.rst

- [x] I have tried with the latest pre-release version and I still can reproduce the issue.

##### Tribler version/branch+revision: #####
next/a85bf00

##### Operating system and version: #####
ubuntu zesty 17.04

##### Expected behavior: #####
tribler allow connections to trackers http://bt2.archive.org:6969/announce and http://bt1.archive.org:6969/announce (or any other tracker)
 
##### Actual behavior: #####
status: Not Working, and though the status of torrents in question are 'Downloading" no data is downloaded.

##### Steps to reproduce the behavior: #####
1) download torrent from https://archive.org/details/rantmedia-newsreal-128kbps
2) open in tribler
3) go to "Trackers" tab for that torrent while it is downloading
4) observe "Not Working" in all trackers.  only DHT is listed as Working.

also observe that the connection *to* archive.org 's tracker is open / not blocked by something on the network :
telnet bt2.archive.org 6969
Trying 207.241.231.226...
Connected to bt2.us.archive.org.
Escape character is '^]'.

moving away ~/.Tribler doesn't change this.

##### Relevant log file output: #####
(no logs)

**Comment by themusicgod1** [comment]
duplicate of  #2681 by the looks :(

**Comment by synctext** [comment]
to document: we only support UDP trackers. Sadly this is extremely difficult to fix, this prevents many Tor-like abuse scenarios.
